stable/redis-ha/README.md

```diff
@@ -41,46 +41,32 @@ $ helm delete <chart-name>
 | `nodeSelector`                   | Node labels for pod assignment                                                                                               | {}                                                        |
 | `tolerations`                    | Toleration labels for pod assignment                                                                                         | []                                                        |
-| `podAntiAffinity.sentinel`       | Antiaffinity for pod assignment of sentinels, `hard` or `soft`                                                               | `soft`                                                    |
 | `podAntiAffinity.server`         | Antiaffinity for pod assignment of servers, `hard` or `soft`                                                                 | `soft`                                                    |
-| `servers.serviceType`            | Set to "LoadBalancer" to enable access from the VPC                                                                          | ClusterIP                                                 |
-| `servers.annotations`            | See Appliance mode                                                                                                           | ``                                                        |
-| `rbac.create`                    |  whether RBAC resources should be created                                                                                    | true                                                      |
-| `serviceAccount.create`          | whether a new service account name that the agent will use should be created.                                                | true                                                      |
-| `serviceAccount.name`            | service account to be used.  If not set and serviceAccount.create is `true` a name is generated using the fullname template. | ``                                                        |
+| `annotations`                    | See Appliance mode                                                                                                           | ``                                                        |
+| `redis.config`                   | Valid redis config options can be added prior to install and will be applied to each server                                  | see values.yaml                                           |
+| `sentinel.config`                | Valid sentinel config options can be added prior to install and will be applied to each server                               | see values.yaml                                           |
 
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```bash
 $ helm install \
-  --set redis_image=quay.io/smile/redis:4.0.6r2 \
+  --set image=quay.io/smile/redis \
```

Comment:
Probably a good idea to also change this example in the README to the official Redis docker image - instead of the older Quay one - to avoid confusion.